server/graphql/v1/mutations/collectives.js

```diff
@@ -313,6 +319,13 @@ export function editCollective(_, args, req) {
     newCollectiveData.countryISO = location.country;
   }
 
+  // Set private instructions value
+  if (args.collective.privateInstructions) {
+    newCollectiveData.data = {
+      privateInstructions: args.collective.privateInstructions,
+    };
```

Comment:
This will erase any existing data from the collective. We need to do:

```es6
    newCollectiveData.data = {
      ...collective.data,
      privateInstructions: args.collective.privateInstructions,
    };
```

Reply:
It should be good now.